### DISCUSSION TITLE
[Question]

### DISCUSSION THREAD

**Discussion by Ruineie** [patrykandpatrick/vico#285] [Questions]
Library information:
Version: 1.6.6
Compose
Question:
I am trying to replicate an component used in google health. It has a column chart for exercise in a given week or say 7 days . The chart is scrollable. When you scroll the chart it will introduce a new week. I want to do some modification on this. I can scroll it freely by any day, one, two and so on. There are only 7 days dispalyed at a time. Can this be implemented in current version? Thank you.

**Comment by Gowsky** [comment]
Hello @Ruineie, are you able to provide a more descriptive example or a visual example of what you want to achieve?
